services/thermal_mgr/thermal_mgr.c

```diff
@@ -43,18 +43,38 @@ void initThermalSystemManager(lm75bd_config_t *config) {
 
 error_code_t thermalMgrSendEvent(thermal_mgr_event_t *event) {
   /* Send an event to the thermal manager queue */
+  xQueueSend(thermalMgrQueueHandle, event, 1);
 
   return ERR_CODE_SUCCESS;
 }
 
 void osHandlerLM75BD(void) {
   /* Implement this function */
+  thermal_mgr_event_t interup;
+  interup.type = InteruptSqn;
+  thermalMgrSendEvent(&interup);
+
 }
 
 static void thermalMgr(void *pvParameters) {
   /* Implement this task */
+
+  float temp;
+  thermal_mgr_event_t buff;
+
   while (1) {
-    
+   xQueueReceive(thermalMgrQueueHandle, &buff, 1);
```

Comment:
We would want the thermal manager to not run until an event is received. Use portMAX_DELAY  to indefinitely block the task until it receives an event.

Reply:
changing to xQueueReceive(thermalMgrQueueHandle, &buff, portMAXDELAY);